### ISSUE TITLE
Loss goes to nan - custom dataset

### ISSUE THREAD

**Issue by nm9404** [pythonlessons/TensorFlow-2.x-YOLOv3#130]
Hey!! :D, first of all, thanks for this amazing implementation of YOLO, has been really helpful to perform some experiments on object detection that I was unable to do as I no longer have tensorflow v.1

I opened this issue because, I'm getting some nan loss values drung training with my custom dataset (specifically after 5 - 6 epochs of transfer learning, I haven't also tried training from scratch), this behaviour doesn't happen on the MNIST example that is provided with the repository.

My custom dataset has only one class and it looks like prob loss converges fast (I attached a tensorboard screenshot below). To solve this I tried default learning rate 1e-4 and 5e-5 to see if it was because of large gradients that it was getting this kind of behaviour but I still get NaN values after some epochs.

I also verified that each object of my dataset had at least one bounding box with the right format, so... I thought it could relate to something on any of the loss functions, but it also may due to something I might be doing wrong on my end... so I would really appreaciate any kind of insight around this

![image](https://user-images.githubusercontent.com/8475509/106543799-6ed18000-64d4-11eb-80e4-303f08d2697a.png)
